### PR TITLE
Testing suite, minor cleanup.

### PR DESCRIPTION
**Pull request by Oman395**
Got bored, looked at this repo, so here we are. I created a testing suite, fairly simple; it just does one or two requests for each API call, and then checks for an http status of 200. I also cleaned up a couple things; for the love of god, don't comment out legacy code, git literally tracks everything done to the files. And song.js had a random console.log for some goddamn reason.

### CODE REVIEW

**Comment by Oman395** [comment]
I could probably set up the error detection to have a set of responses that are 100% correct, and check off that, but it's getting late and I'm tired. Also could probably do checks on the webpages, but, well, see above.

File: api/song.js [hunk 1]
```diff
@@ -11,46 +11,7 @@ module.exports = async (app, req, res) => {
         if (err) return res.send('-1')
         else if (body < 0) return res.send(false)
         request.get('https://www.newgrounds.com/audio/listen/' + songID, function(err2, resp2, song) {
-            console.log(resp2.statusCode)
             return res.send(resp2.statusCode == 200)
         })
     })
-}
```
Comment (Oman395) [inline api/song.js:18]: typo, need to fix

**Comment by GDColon** [comment]
Forgive me for having a brain the size of a nickel but I'm not really sure I understand the purpose of this. To me it just looks like it's sending a request to every endpoint in the GD servers which will inevitably trigger a local IP ban.

**Comment by Oman395** [comment]
> Forgive me for having a brain the size of a nickel but I'm not really sure I understand the purpose of this. To me it just looks like it's sending a request to every endpoint in the GD servers which will inevitably trigger a local IP ban.

It's a testing script-- so when making changes to the server, you can quickly run it, and ensure that the code didn't screw something up

**Pull request closed without merging**